bin/compile

```diff
@@ -63,24 +63,24 @@ echo "Downloading Chrome" | indent
 ZIP_URL="https://storage.googleapis.com/chrome-for-testing-public/$VERSION/linux64/chrome-linux64.zip"
 ZIP_LOCATION="$INSTALL_DIR/chrome.zip"
 curl --silent --show-error --fail --retry 3 --retry-connrefused --connect-timeout 10 -o "${ZIP_LOCATION}" "${ZIP_URL}" | indent
-unzip -q -j -o "$ZIP_LOCATION" -d "$BIN_DIR" | indent
+unzip -q -o "$ZIP_LOCATION" -d "$BIN_DIR" | indent
 rm -f "$ZIP_LOCATION"
 
 echo "Downloading Chromedriver" | indent
 ZIP_URL="https://storage.googleapis.com/chrome-for-testing-public/$VERSION/linux64/chromedriver-linux64.zip"
 ZIP_LOCATION="$INSTALL_DIR/chromedriver.zip"
 curl --silent --show-error --fail --retry 3 --retry-connrefused --connect-timeout 10 -o "${ZIP_LOCATION}" "${ZIP_URL}" | indent
-unzip -q -j -o "$ZIP_LOCATION" -d "$BIN_DIR" | indent
+unzip -q -o "$ZIP_LOCATION" -d "$BIN_DIR" | indent
 rm -f "$ZIP_LOCATION"
 
 source "$BUILDPACK_DIR/bin/install-chrome-dependencies"
 
 echo "Adding executables to PATH" | indent
 mkdir -p "$BUILD_DIR/.profile.d"
-echo "export PATH=\$HOME/.chrome-for-testing/bin:\$PATH" >> "$BUILD_DIR/.profile.d/chrome-for-testing.sh"
+echo "export PATH=\$HOME/.chrome-for-testing/bin/chrome-linux64:\$HOME/.chrome-for-testing/bin/chromedriver-linux64:\$PATH" >> "$BUILD_DIR/.profile.d/chrome-for-testing.sh"
```

Comment:
Should we remove the `bin/` sub-nesting now too? (The chrome archive contains more than just a binary)

Also, should these PATH usages re-use the env vars (`$INSTALL_DIR` or `$BIN_DIR`) rather than duplicating the filepath?